cmd/exporter.go

```diff
@@ -191,25 +71,28 @@ func Run(ctx *cli.Context) error {
 	onShutdown := make(chan os.Signal, 1)
 	signal.Notify(onShutdown, syscall.SIGINT, syscall.SIGTERM, syscall.SIGABRT)
 
-	stopPolling := make(chan bool)
-	c.pollProjects(stopPolling)
+	untilStopSignal := make(chan bool)
+	pipelinesOnInit := make(chan bool)
+	c.orchestratePolling(untilStopSignal, pipelinesOnInit)
+	// get immediately some data from the latest executed pipelines, if configured to do so
+	pipelinesOnInit <- cfg.OnInitFetchRefsFromPipelines
 
 	// Configure liveness and readiness probes
 	health := healthcheck.NewHandler()
-	if !cfg.Gitlab.SkipTLSVerify {
+	if !cfg.Gitlab.SkipHealthCheck {
```

Comment:
I think it would be nice to have a switch with more precise log.Warn here instead of solely replacing the statement on `cfg.Gitlab.SkipTLSVerify`

Reply:
Ah the message was not updated... thanks 👍 
Changed to ```
"GitLab health check has been disabled. Readiness checks won't be operated."```

Reply:
thanks! we are still missing the case `cfg.Gitlab.SkipTLSVerify == true` though :)